rlog/state-separation.mdx

```diff
@@ -0,0 +1,1255 @@
+---
+title: 'Nescience - A user-centric state separation architecture'
+date: 2024-08-07 12:00:00
+authors: moudy
+published: false
+slug: Nescience-A-state-separation-architecture
+categories: research
+
+
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+---
+
+Nescience, a user-centric state separation architecture.
```

Comment:
"Nescience: A user-centric state separation architecture."

Seems redundant - it's the same as the title and will appear directly under it. If we want to include it, we should stylise it thusly:

"Nescience: A user-centric state-separation architecture."